includes/manage_pages.inc

```diff
@@ -794,6 +798,12 @@ function islandora_paged_content_add_pages($repository, $config, $pages_director
   $parent = $config['pid'];
   $object->relationships->add(FEDORA_MODEL_URI, 'hasModel', $config['model']);
 
+  if ($config['ignored_derivs']['ocr']) {
+    islandora_paged_content_set_relationship($rels_ext, ISLANDORA_RELS_EXT_URI, 'generate_ocr', 'FALSE', TRUE);
+  }
+  if ($config['ignored_derivs']['hocr']) {
+    islandora_paged_content_set_relationship($rels_ext, ISLANDORA_RELS_EXT_URI, 'generate_hocr', 'FALSE', TRUE);
+  }
```

Comment:
Same question about if they don't have `islandora_ocr` installed, these keys may not be set properly.